source/tyr/tyr/resources.py

```diff
@@ -107,7 +107,36 @@ def post(self, instance_name):
         return {'message': 'OK'}, 200
 
 
-class PoiType(flask_restful.Resource):
+def _validate_poi_types_json(poi_types_json):
```

Comment:
Since this function does more verifications than the generic function 'validate', comments to explain are always helpful